Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
Hi Nicholas and Sophie,
Nich, the solution you propose was the first Martin suggested. We werent able to make it works so we used friendly_id.
I'll look at the alternative way to see if I can make it work.
Hey, I want to thank you for the organisation, it was awesome.